WATOLINK/bci-gaming
# Isaac game music and background

## ClickUp ticket link
[Game Music](https://app.clickup.com/t/8685upafc)
[Endless Background](https://app.clickup.com/t/8685qf3bn)

## Implementation description
- Game Music: 
   -  Plays music on load. The object will not be destroyed when transitioning into different scenes (i.e the background music will continue playing across different scenes)
   - It is a prefab that can just be dragged into the scene.

- Endless Background:
   - Background tracks the camera position. If the camera is moving too far to the right, the background moves to the right (same as left)  
   - It is a prefab that can just be dragged into the scene.

## Steps to test
1. Go under Assets >> Prefabs and look for "Background" and "GameMusic"
2. Drag the Prefabs into the scene.
3. See if music plays and the background feels endless

## What should reviewers focus on?

Background Music: When you navigate into a different scene the music should still play. When you navigate back to the menu, there should only be one BackgroundMusicGameManager playing music. 

Background: Making sure the endless effect is smooth (no jittery movement)

## Checklist
